### PR TITLE
add ask answer only when env-file does not have one

### PR DESCRIPTION
**Pull request by carolynhu**
https://github.com/rancher/rancher/issues/10164

### CODE REVIEW

File: lookup/fileenv.go [hunk 1]
```diff
@@ -119,6 +118,23 @@ func NewFileEnvLookup(file string, parent config.EnvironmentLookup) (*FileEnvLoo
...
+	for key, question := range questions {
+		if _, ok := combMap[key]; ok {
+			continue
+		}
+		answer := ask(question)
+		if answer != "" {
+			variables[key] = answer
+		}
+	}
+
```
Comment (StrongMonkey) [inline lookup/fileenv.go:137]: Can you give a brief description on why this change would fix the issue?

Comment (carolynhu) [reply lookup/fileenv.go:137]: @StrongMonkey you first lookup rancher-compose to gather all the questions, then lookup env-file to parsing those key-value pairs, after that, iterate over all questions to see whether that question key has a value being parsed, if not, simply ask user to provide one

**Comment by carolynhu** [comment]
Have tested on both compose-executor and CLI, it works.

**Comment by StrongMonkey** [comment]
LGTM
